rust/src/aoc/day_02.rs

```diff
@@ -1,26 +1,49 @@
 use super::input::INPUT_PATH;
+use anyhow::{anyhow, Error, Result};
+use itertools::Itertools;
 use std::fs;
 use std::path::Path;
 use std::str::FromStr;
 
 static DAY: i16 = 2;
 
+#[derive(Default)]
```

Comment:
If there's a better `Default` setup, you can `impl Default` yourself rather than derive it. Most things go for the "emptiest" default they can when you derive it, so e.g. all numeric values are set to 0.